frontend/src/app/services/snack-bar.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+import { MatSnackBar } from "@angular/material/snack-bar";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SnackBarService {
+
+  constructor( private snackBar: MatSnackBar) { }
+
+  error(message: string){
+    this.snackBar.open(message, 'OK', {
+      duration: 5000,
+      panelClass: ['snack-error']
+    });
+  }
+
+  success(message: string){
+    this.snackBar.open(message, 'OK', {
+      duration: 5000,
+      panelClass: ['snack-success']
```

Comment:
`snack-success` and `snack-error` are your custom classes. you can set background green/red for them, etc.